tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 
 envlist =
-    flake8
+    flake8, mypy
```

Comment:
Any reason not to add it on its own line?

Reply:
No; I just mentally grouped the lint-related environments, and that leaked out here.

Reply:
split in two